docs/user/configuring-agent.md

```
@@ -135,6 +136,115 @@
+journalctl -u flightctl-agent -f
+
+# Filter for TPM messages
+journalctl -u flightctl-agent | grep -i tpm
+```
+
+Enable debug logging for detailed TPM information:
+
+```yaml
+# /etc/flightctl/config.yaml
+log-level: debug  # or trace for maximum detail
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)